selfdrive/car/toyota/interface.py

```diff
@@ -28,8 +28,14 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_TOYOTA_ALT_BRAKE
 
     if candidate in ANGLE_CONTROL_CAR:
-      ret.dashcamOnly = True
       ret.steerControlType = SteerControlType.angle
+      for fw in car_fw:
+        # Detect if we can control the car with torque (cars made in Japan)
+        if fw.ecu == "eps" and not fw.fwVersion == b'8965B42371\x00\x00\x00\x00\x00\x00':
```

Comment:
this blacklist seems too specific? I doubt all 23 Rav4's with angle will have this FW, but they can potentially fingerprint as torque cars now

Reply:
I should not have placed that `not` there, this is the FW for the torque cars.